lib/Echidna/Campaign.hs

```diff
@@ -121,7 +122,7 @@ data Campaign = Campaign { _tests       :: [(SolTest, TestState)]
 instance ToJSON Campaign where
   toJSON (Campaign ts co gi _ _ _ _) = object $ ("tests", toJSON $ mapMaybe format ts)
     : ((if co == mempty then [] else [
-    ("coverage",) . toJSON . mapKeys (`showHex` "") $ DF.toList <$> co]) ++
+    ("coverage",) . toJSON . mapKeys (("0x" ++) . (`showHex` "") . keccak) $ DF.toList <$> co]) ++
```

Comment:
Is this JSON deserialized somewhere?

Reply:
What do you mean exactly? Where this JSON is used after it is saved?

Reply:
Yes

Reply:
I don't know, perhaps @agroce is using it to capture Echidna usage in his experiments.

Reply:
got it, I guess we can proceed since this is undocumented interface, whoever is using that needs to adjust though